charts/prefect-server/templates/server-deployment.yaml

```diff
@@ -128,18 +131,19 @@ spec:
           {{- with .Values.server.containerSecurityContext }}
           securityContext: {{- toYaml . | nindent 12 }}
           {{- end }}
+          {{- if .Values.server.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: /api/health
+              port: {{ .Values.service.targetPort }}
+          {{- toYaml .Values.server.livenessProbe.config | nindent 12 }}
+          {{- end }}
+          {{- if .Values.server.readinessProbe.enabled }}
           readinessProbe:
-            exec:
-              command:
-                - python
-                - -c
-                - "from prefect.server.database import get_db_connection; import asyncio; asyncio.run(get_db_connection())"
```

Comment:
@zzstoatzz is this something we should be using here??

Reply:
no i think we should remove that, didn't mean to leave it in

Reply:
cool, gone!